controls/radtreeview/features/ui-virtualization.md

```diff
@@ -20,49 +20,40 @@ The following tutorial shows how to bind to a collection of business objects and
 
 Here is a simple __RadTreeView__ declaration.
 
-#### __XAML__
+#### __[XAML] Example 1: Bind RadTreeView to large number of items__
 
 {{region radtreeview-features-ui-virtualization_0}}
-	<UserControl.Resources>
-	
-	    <sampleData:RadTreeViewSampleData x:Key="DataSource"/>
-	
+	<UserControl.Resources>	
+	    <sampleData:RadTreeViewSampleData x:Key="DataSource"/>	
 	    <DataTemplate x:Key="TeamDataTemplate">
 	        <TextBlock Text="{Binding Name}"/>
-	    </DataTemplate>
-	
+	    </DataTemplate>	
 	</UserControl.Resources>
 	
-	<Grid x:Name="LayoutRoot" Background="White">
-	
-	    <telerik:RadTreeView x:Name="radTreeView">            
-	        <telerik:RadTreeViewItem Header="League A">
-	
-	            <telerik:RadTreeViewItem Header="Teams" 
-	                ItemsSource="{Binding Source={StaticResource DataSource}, Path=VeryLargeDataSource}"
-	                ItemTemplate="{StaticResource TeamDataTemplate}"/>
-	
-	        </telerik:RadTreeViewItem>
-	    </telerik:RadTreeView>
-	
+	<Grid x:Name="LayoutRoot">	

```

Comment:
If there is another article, defining the RadTreeViewSampleData class, we should link to it.